benchmark/src/main/java/com/amazon/opendistroforelasticsearch/sql/benchmark/utils/load/elasticsearch/ElasticsearchDataLoader.java

```diff
@@ -38,16 +40,26 @@ public void loadData(final DataFormat data) throws Exception {
     }
     String commands = "cd " + ((ElasticsearchDataFormat) data).getDataPath();
 
+    // Add table mappings
     for (String tableName : TpchSchema.schemaMap.keySet()) {
```

Comment:
Does this need to be switched to ElasticsearchDataFormat . getTableDataFilesList?

Reply:
No, the mapping is fixed and it won't change according to SF. just one file per table as `<tablename>_mappings.json`. this file just has info about all fields and it's data type for all 8 tables. So we do not need a data files list.